scipy/sparse/csgraph/setup.py

```diff
@@ -20,6 +20,10 @@ def configuration(parent_package='', top_path=None):
     config.add_extension('_min_spanning_tree',
          sources=['_min_spanning_tree.c'],
          include_dirs=[numpy.get_include()])
+
+    config.add_extension('_matching',
+         sources=['_matching.c'],
```

Comment:
It looks like `scipy/sparse/csgraph/_matching.c` will need to be added to `.gitignore`, although this more of a refinement comment really